cplus.py

```
@@ -0,0 +1,8 @@
+// Optimized loop for performance-critical task
```

Ruff (0.8.2)

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Invalid assignment target

---

SyntaxError: Expected 'in', found name

---

SyntaxError: Expected ':', found newline

---

SyntaxError: Expected an indented block after `for` statement

---

SyntaxError: Expected ')', found name

---

SyntaxError: Expected ':', found '='

---

SyntaxError: Expected a statement

---

SyntaxError: Expected an expression

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected '}', found 'for'

---

SyntaxError: Expected 'in', found newline

---

Statement ends with an unnecessary semicolon

(E703)